cmd/run.go

```diff
@@ -229,6 +257,10 @@ func init() {
 	RunCmd.Flags().StringVarP(&protocol, "protocol", "", "http", "tells Dapr to use HTTP or gRPC to talk to the app. Default is http")
 	RunCmd.Flags().String("redis-host", "localhost", "the host on which the Redis service resides")
 	RunCmd.Flags().String("placement-host", "localhost", "the host on which the placement service resides")
+	if runtime.GOOS != "windows" {
+		// Deamon feature does not work on Windows.
+		RunCmd.Flags().BoolVarP(&isDaemon, "daemon", "d", false, "starts dapr and app in the background")
+	}
 
```

Comment:
New usage should be added in this method to describe new ways dapr run can be called.

Reply:
Added "Long" attribute. See new version.